### DISCUSSION TITLE
example for createSourceEventStream from 'graphql/subscription'

### DISCUSSION THREAD

**Discussion by Elaman19** [graphql/graphql-js#4232] [Q&A]
Hi! Can you write example how to use createSourceEventStream from 'graphql/subscription'?
Thanks.

**Comment by robzhu** [comment]
Is a createSourceEventStream example all you need? Would a more comprehensive end-to-end example be more useful?

**Comment by Elaman19** [comment]
Yes, i need more comprehensive end-to-end example. Can you write or give a link?

**Comment by chaitanya11** [comment]
Does anyone has anything on this issue ?
